packages/vscode-extension/src/webview/components/NetworkFiltersDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import "./shared/Dropdown.css";
+import { DropdownMenuRoot } from "./DropdownMenuRoot";
+import CheckIcon from "./icons/CheckIcon";
+
+interface NetworkFiltersDropdownProps {
+  children: React.ReactNode;
+  disabled?: boolean;
+}
+
+function NetworkFiltersDropdown({ children, disabled }: NetworkFiltersDropdownProps) {
+  const [checked, setChecked] = React.useState(true);
+  return (
+    <DropdownMenuRoot>
+      <DropdownMenu.Trigger asChild disabled={disabled}>
+        {children}
+      </DropdownMenu.Trigger>
+      <DropdownMenu.Portal>
+        <DropdownMenu.Content className="dropdown-menu-content">
+          <DropdownMenu.CheckboxItem checked={checked} onCheckedChange={setChecked}>
+            <DropdownMenu.ItemIndicator>
```

Comment:
Do all of these dropdowns do nothing?
They're all controlled by the same `checked` variable, and it doesn't seem to be used anywhere else.